libs/components/theme/src/lib/theming/theme-brand.spec.ts

```diff
@@ -115,6 +140,36 @@ describe('Theme brand', () => {
       expect(brand.name).toBe('custom');
       expect(brand.version).toBe('2.0.0');
       expect(brand.hostClass).toBe('custom-host-class');
+      expect(brand.sriHash).toBeUndefined();
+    });
+
+    it('should deserialize brand with SRI correctly', () => {
+      const sriHash = 'sha384-abc123def456ghi789jkl012mno345pqr678stu901vwx234yz567890abcdef';
+      const brand = SkyThemeBrand.deserialize({
+        name: 'rainbow',
+        version: '1.0.0',
+        sriHash: sriHash,
```

Comment:
Since the variable is the same name as the property being assigned, use the shorthand syntax for object property assignemnt. Do the same in other tests in this same file.